dojo/templates/dojo/view_eng.html

```diff
@@ -649,14 +649,16 @@ <h4 class="pull-left">
                             {% for finding in findings %}
                                 <tr class="{% if finding.active %}active_finding{% else %}inactive_finding{% endif %}">
                                     <td><a href="{% url 'view_finding' finding.id %}">{{ finding.title }}</a></td>
-                                    <td>{% if finding.severity == "Critical" or finding.severity == "High" %}
-                                        <span class="text-error">
-                                    {% else %}<span>{% endif %}{{ finding.severity }}</span></td>
+                                    <td>
+                                        <span class="label severity severity-{{ finding.severity|safe }}">
```

Comment:
Haha, that's ok! :smile: 

That's a good catch, it's from my initial misunderstanding of what the safe function did :roll_eyes: 

I'll change that and go through any other uses of |safe in my commits to make sure they're actually needed!